doc/Programming-guide/Key-platform-features/Notifications/index.md

```diff
@@ -151,45 +142,38 @@ Assuming that you have created custom endpoint groups from the [Using endpoint g
     </tr>
 </table>
 
-#### Sending notifications
-To send a notification, you can issue the [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Notifications/sendNotification) request or Administration UI as described below.
+### Send notifications
 
-Do the following steps to send a notification:
+To send a notification, use the [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Notifications/sendNotification) or use the Administration UI:
 
-1. In the **Notification topics**, click the mail icon next to the appropriate notification topic.
-2. In the **Send notification** window, create a notification either by using the **Notification body** [record form]({{root_url}}Administration-guide/Tenants-and-applications-management/#record-form) or by uploading the data in the JSON format from a file.
+1. Select the topic from the **Notification topics** page of the application and click **Send notification**.
 
-> **NOTE:** The contents of the file should match the corresponding notification schema. And if **Endpoint KeyHash** field is empty will be sent a broadcast notification.
+2. On the **Notification details** page, create a notification either by using the **Notification body** [record form]({{root_url}}Administration-guide/Tenants-and-applications-management/#record-form) or by uploading a JSON file.
+The data structure of your JSON file must match the corresponding notification schema.
+
+	![Send Notification](images/send_notification.png)
 
-![Send Notification](images/send_notification.png)
 3. Click **Send** to send the notification.
 
-The file with the following contents will match the default sandbox notification schema.
+Below is an example of the uploaded file contents that will match the default Sandbox notification schema.
 
 ```
 {"message": "Hello from Kaa!"}
 ```
 
-#### Notification pipelines
+A notification will be queued for delivery until the time you specified in the **Expires at** field of the **Notification details** page.
+If you leave this field blank, the message will be queued until it is delivered.
```

Comment:
This is not true, default notification ttl is 1 week. This is hardcoded parameter in NotificationServiceImpl. And yes, i will open a ticket for this :-)